plot_friends.py

```diff
@@ -86,5 +69,13 @@ def friends():
     plt.legend(loc='upper left', ncol=2)
     plt.show()
 
+    print('\nSome statistics based on the data: ', maxdays)
+    stats = []
+    stats.append(["Total Friends", cumulative_friends[-1]])
+    max_frnd_day = xaxis[frndcount.index(max(frndcount))]
+    stats.append(["Maximum Friends made on", max_frnd_day.date()])
+    stats.append(["Maximum Friends made in month", max_frnd_day.strftime('%B')])
```

Comment:
Let's say I made most friends on 28 Feb 2018.
I made the most friends in July.
Currently, the script returns `February`, I think it should return `July`.

Reply:
sure !!